app/plugins/user_interface/websocket/index.js

```diff
@@ -537,6 +537,12 @@ function InterfaceWebUI(context) {
 			});
 
 			connWebSocket.on('getBrowseSources', function (date) {
+				
+				//Refresh cacheAlbumList			
+				var mpdPlugin = self.commandRouter.pluginManager.getPlugin('music_service', 'mpd');
+				mpdPlugin.listAlbums();
```

Comment:
This will be a problem, since this function is called every 4 seconds when user sits on my music page